components/grid-col/src/index.js

```diff
@@ -14,20 +14,19 @@ const colValues = {
 };
 
 const widthFromProps = spacing.withWidth({ noDefault: true });
-const desktopWidthFromProps = spacing.withWidth({ mediaQuery: MEDIA_QUERIES.DESKTOP, noDefault: true });
+const desktopWidthFromProps = spacing.withWidth({
+  mediaQuery: MEDIA_QUERIES.DESKTOP,
+  noDefault: true,
+});
 
 function setGrowShrink(style) {
   const hasAutoWidth = [undefined, 'auto'].includes(style.width);
 
   // No explicit width means auto, so grow/shrink should be set
-  return Object.assign(
-    {},
-    style,
-    {
-      flexGrow: hasAutoWidth ? 1 : 0,
-      flexShrink: hasAutoWidth ? 1 : 0,
-    },
-  );
+  return Object.assign({}, style, {
```

Comment:
not keen on this either

previous version made it clearer what objects where being merged together
this new version is harder to parse